src/components/VideoList/VideoList.component.jsx

```diff
@@ -3,9 +3,9 @@ import React from 'react';
 import VideoItem from '../VideoItem';
 import { GridContainer } from './VideoList.styles';
 
-const VideoList = ({ videos, index }) => {
+const VideoList = ({ videos }) => {
   const renderedVideos = videos.map((video) => {
-    return <VideoItem key={index} video={video} videos={videos} />;
+    return <VideoItem key={video.id.videoId} video={video} videos={videos} />;
```

Comment:
if for some reason there's no `video` or `id` to get `videoId` prop, you could get unexpected errors, I suggest using the index provided by map instead, as it was previously.

Any reason for the change?

Reply:
I got this error
<img width="775" alt="Screen Shot 2021-03-22 at 22 04 12" src="https://user-images.githubusercontent.com/74254046/112091207-8afbb000-8b5a-11eb-8a36-6f5b9b450a8e.png">